final-proj-code.Rmd

```diff
@@ -69,6 +78,10 @@ mean_row <- function(df){
   output
 }
 
+# Area of improvement: I think using rowwise analyses is a brilliant idea for ipsatizing data, but I'm confused why you're not using the rowwise function (discussed here https://fp-2021.netlify.app/slides/w4p2.html#41)? It seems like using that function could simplify your code and effectively replace your use of pivot_longer and pivot_wider. 
```

Comment:
Great feedback! I think I was actually the one that suggested `pivot_longer()` but this is even better!